.travis.yml

```diff
@@ -189,4 +190,3 @@ jobs:
         - sudo cp odo /usr/bin
         - export KUBERNETES=true
         - travis_wait make test-cmd-devfile-catalog
-        - travis_wait make test-cmd-devfile-watch
```

Comment:
This make target runs on Kubernetes cluster. It seems unfortunately it has been deleted, no ?